apis_core/apis_entities/urls.py

```diff
@@ -88,4 +89,5 @@ def to_url(self, value):
         "entity/<entitytocontenttype:contenttype>/",
         include(entity_patterns),
     ),
+    path("autocomplete", EntitiesAutocomplete.as_view(), name="autocomplete"),
```

Comment:
```suggestion
    path("autocomplete/", EntitiesAutocomplete.as_view(), name="autocomplete"),
```

not working in my current setup without the trailing slash. Though it worked before?
However, trailing slashes in all other routes, so should be there anyway